### DISCUSSION TITLE
stat-computing-r-python/part-tools/02-setting-up-computer

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#3] [General]
# stat-computing-r-python/part-tools/02-setting-up-computer

https://srvanderplas.github.io/stat-computing-r-python/part-tools/02-setting-up-computer.html

**Comment by ggergen2** [comment]
The link to Jupyter does not work.

**Reply by srvanderplas** [reply]
Fixed in [90e996c](https://github.com/srvanderplas/stat-computing-r-python/commit/90e996c2f9c32a3359deb687ecbb0e2091c34419). Thanks!

**Comment by bbaum44** [comment]
the "these instructions" link doesn't work

**Reply by srvanderplas** [reply]
Fixed in https://github.com/srvanderplas/stat-computing-r-python/commit/68858899a3ffef188e6a2254f11c45aa450396e8. Thanks!
